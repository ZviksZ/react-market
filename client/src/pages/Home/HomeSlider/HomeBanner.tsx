import * as React from 'react'
import photo2 from '../../../assets/img/2.jpg'
import s from './HomeBanner.module.scss'

export const HomeBanner: React.FC = () => {
	return (
		<>
			<div className={s.mainBanner} style={{ backgroundImage: 'url(' + photo2 + ')' }}></div>
		</>
	)
}
