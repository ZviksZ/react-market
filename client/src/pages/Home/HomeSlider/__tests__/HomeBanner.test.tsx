// @ts-nocheck
import * as React from 'react'
import { render } from '@testing-library/react'
import { HomeBanner } from '../HomeBanner'

describe('testing HomeBanner component', () => {
	it('should be correct snapshot', () => {
		const { container } = render(<HomeBanner />)

		expect(container).toMatchSnapshot()
	})
})
