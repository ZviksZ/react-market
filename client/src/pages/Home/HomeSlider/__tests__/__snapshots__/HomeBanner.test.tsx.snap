// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing HomeBanner component should be correct snapshot 1`] = `
<div>
  <div
    class="mainBanner"
    style="background-image: url(2.jpg);"
  />
</div>
`;
